app/src/main/res/layout/download_management.xml

```diff
@@ -13,9 +13,8 @@
     android:layout_gravity="center_vertical"
     android:gravity="center_vertical"
     android:orientation="vertical"
-    android:paddingBottom="75dp"
-    android:text="@string/no_downloads_here"
```

Comment:
Why have you deleted the text ?

Reply:
The text in the downloading tab was also hardcoded. so I moved it as well.

Reply:
Sorry, didn't get you. It was hardcoded, then why delete it ?

Reply:
just a second. you are right. there is a problem. just let me check. give me a minute